static/app/views/issueDetails/actions/index.tsx

```diff
@@ -620,3 +644,9 @@ const ResolvedActionWapper = styled('div')`
   gap: ${space(1)};
   align-items: center;
 `;
+
+const ReasonBanner = styled('div')`
```

Comment:
we probably want to keep consistent with how we display links to releases (gray instead of blue)

Reply:
<img width="436" alt="image" src="https://github.com/user-attachments/assets/0460ff6d-45a6-404d-8e50-55d9c828b964" />

Fixed for releases and commits as well. Went with the green from the rest of the text since gray also looked a bit off.